donkeycar/parts/camera.py

```diff
@@ -1,8 +1,10 @@
 import logging
 import os
 import time
+import threading
 import numpy as np
 from PIL import Image
+import mss
```

Comment:
Please have a look at the failed tests, you need to add the python packages into setup.cfg in the right place. Likely we only want to support that in the PC installation but not on the robot.